stmocli/cli.py

```diff
@@ -40,6 +42,12 @@ def track(conf, query_id, file_name, redash_api_key):
             urljoin(redash.API_BASE_URL, url_path)
         )
 
+        if not file_name:
```

Comment:
Why include the query id in the name?

Reply:
I was using it for a couple reasons:

* something to copy-paste into the URL bar to load something in re:dash (which argues for a `stmocli view` command)
* as part of the devtools-dashboard-refresh task I'm working on, I'm tracking the query IDs of refreshed queries in a spreadsheet, and this made the relationship between the spreadsheet row and the query sql immediately available

So it was the naming pattern that I was using anyway but as long as there's _a_ default I don't think I have strong feelings.

Reply:
It's not a strong opinion, but prepending the query_id feels like duplicated information to me. Agree we should have a `stmocli view` command.